### ISSUE TITLE
Confirmation:  Successful:  RO Routing:  Get next case

### ISSUE THREAD

**Issue by nikitarockz** [department-of-veterans-affairs/caseflow#473]
As a Caseworker, I want to be able to quickly move to the next case or stop working after successfully submitting an End Product or RO Routing so that I can continue working case or move onto other tasks.

**Acceptance Criteria**

1. Verify that after successfully submitting an RO Routing, there is a “Next Case” button.
1. Verify that after successfully submitting an RO Routing, there is a “Back to Home” link. (??? Not sure if this is what is wanted.)

**Comment by KHarshawat** [comment]
@nikitarockz I'm not sure this issue accuratley captures the scope of what needs to be developed or if there are more issues that need to be created.  The idea of getting to the next case is one that's independent of which confirmation message is displayed. Meanwhile there are multiple permutations of congratulations messages that could be displayed depending on what happened to the EP in earlier steps. There is a lot of logic to be added here.  How do you want to break these up and should we create another issue for the success message?

**Comment by KHarshawat** [comment]
Update: I see that this covers the Rerouted confirmation message: https://github.com/department-of-veterans-affairs/caseflow/issues/422

**Comment by nikitarockz** [comment]
Yeah, this is one of my early ones that was not discussed or reviewed.  I was working from a list Abby had given me and I thought of the negative case of this process.  Again, at that time based on limited information this wasn't discussed.  Based on what I know now, this would be different.



Sent from my Verizon, Samsung Galaxy smartphone


-------- Original message --------

**Comment by nikitarockz** [comment]
@KHarshawat You mentioned that you have the latest RO mockup.  Can you send me that link and I can add them to the RO issues.  I know there are still conversations around how the RO is going to work.  I am not sure if everything has been ironed out.

**Comment by KHarshawat** [comment]
@nikitarockz @lakohl is working on those mocks. She finished them yesterday and should be sharing them today.

**Comment by lakohl** [comment]
@nikitarockz I'm going to go over the RO Routing mocks with Abby this afternoon to make sure they cover all the necessary steps & that the lang is correct then will post them.
@KHarshawat did you do an RO variation on the confirmation page yesterday? If not that's fine, I will include it.

**Comment by KHarshawat** [comment]
@lakohl I do! See below.

![congrats rerouted - full grant](https://cloud.githubusercontent.com/assets/6335546/20801135/dbbd2e6a-b7b5-11e6-9f65-10128f8693a1.png)
![congrats rerouted - partial grant or remand](https://cloud.githubusercontent.com/assets/6335546/20801136/dbc42d00-b7b5-11e6-92d9-6c23a82ce174.png)

And my latest sketch file for editing:

[Caseflow Dispatch (AMC) 20161130.zip](https://github.com/department-of-veterans-affairs/caseflow/files/625121/Caseflow.Dispatch.AMC.20161130.zip)

**Comment by lakohl** [comment]
@KHarshawat Great, thanks!
